src/index.html

```diff
@@ -11,6 +11,44 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      class="container"
+      data-qa="big">
+      <label>
+        <img
+          src="./images/Search.svg"
+          alt="img-req-alt"
+          class="bigImg img"
+        >
+        <input
+          type="text"
+          name="bSearch"
+          data-qa="keypress"
+          class="form big"
+          placeholder="Try &quot;Los Angeles&quot;"
+        >
+      </label>
+    </form>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      class="container"
+      data-qa="small">
```

Comment:
closing brace should be at the same level as opening one - this way your layout is better structured and it is easier to distinguish where ends one tag and the other starts
```suggestion
    <form
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
      method="post"
      class="container"
      data-qa="small"
    >
```